### PR TITLE
Some updates to CONTRIBUTORS in preparation for 1.12

### PR DESCRIPTION
**Pull request by bradcray**
This is a first draft, though I need to poll people and review the CLAs to see who I might be forgetting.

### CODE REVIEW

**Comment by bradcray** [comment]
@konsP -- can you double check that I've spelled your name correctly here?  :)  (I've been known to get much simpler ones wrong at times).  @mppf, if you could double check Damian's name and affiliation, I'd appreciate that as well.

**Comment by konsP** [comment]
@bradcray surname is correct.
Could I also get sth like "added 1 link and wrote 2 lines" next to my name please?

File: CONTRIBUTORS [hunk 1]
```diff
@@ -18,13 +20,12 @@ Ben Harshbarger, Cray Inc.
 Tom Hildebrandt, Cray Inc.
 David Iten, Cray Inc.
 Vassily Litvinov, Cray Inc.
-Simon Lund, Københavns Universitet (University of Copenhagen)
 Tom MacDonald, Cray Inc.
-Phil Nelson, Western Washington University
+Damian McGuckin, Pacific ESI
```
Comment (mppf) [inline CONTRIBUTORS]: I'd put Pacific Engineering Systems International
(spelled out) - otherwise it's fine.

**Comment by bradcray** [comment]
mppf: Done, thanks.
konsP: No you may not -- all contributions are equally worthy here and I don't like to keep score.  :)

**Pull request merged**